src/components/Error/Error.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+
+type Props = {
+  error: string,
+  isHiddenClass: boolean,
+  setIsHiddenClass: (val: boolean) => void,
+};
+
+export const Error: React.FC<Props> = ({
+  error,
+  isHiddenClass,
+  setIsHiddenClass,
+}) => {
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setIsHiddenClass(true);
+    }, 3000);
+
+    return () => {
+      clearTimeout(timer);
+    };
+  }, [setIsHiddenClass, isHiddenClass]);
```

Comment:
```suggestion
  }, [isHiddenClass]);
```
`setIsHiddenClass ` redundant in this case